### DISCUSSION TITLE
CPU and GPU temp

### DISCUSSION THREAD

**Discussion by WalkerJoel576** [seerge/g-helper#3499] [General]
Hi guys, im new here. i just purchased my G16 2024, now using Ghelper . i'd like to know a few things on my CPU/GPU temp

so im not really experienced with these settings. 

while im on silent mode CPU hit 47-53c RPM on 1900-2400 (GPU on standard all factory settings). while gaming on Turbo mode CPU and GPU hit 79-81c 

so my question is the silent mode CPU temp ideal for long term ? im looking to use this laptop for atleast 4-5years , is it possible to hit a 40-45c range while on silent mode ? couldnt quite get it lower while fiddling with the settings. 

and on gaming wise 79-81c feels better cause on AC im hitting 90-95 . what settings can be recommended to keep my CPU n GPU temp 75-80c while gaming ? 

the 79-81c i get was while i played lies of p settings with Resolution : 2560x1600 , Frame rate :165 , Vsync : on, Nvidia DLSS : Performance , Graphic quality preset : medium , Nvidia Super resolution : off ( i find the temp rising quickly with this on ,so i turn it off) 

any advise on this is appreciated. thanks!

**Comment by WalkerJoel576** [comment]
oh and the spec of G16 i own is , U9 185H 32GB RAM, RTX 4070

**Comment by seerge** [comment]
@WalkerJoel576 hello, unless your CPU is going above throttle limit (96C) or GPU is going above 87C you are absolutely fine. Your device is designed to run like that.

40-50C for Silent is also absolutely fine. Don't forget that since you keep your GPU enabled (Standard or Optimized when plugged in) it keeps working and heating internals. You may power off GPU completely by setting (Eco), then laptop should run even cooler.

Thanks

**Reply by WalkerJoel576** [reply]
Noted on that , Thanks for the advice !
